src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -700,10 +701,13 @@ export class SavedObjectsRepository {
    * @property {Array<string>} [options.fields]
    * @property {string} [options.namespace]
    * @property {object} [options.hasReference] - { type, id }
+   * @property {object} [options.aggs] - see ./saved_object_aggs for more insight
```

Comment:
Seems like this file no longer exists. We don't really use these JSDoc comments anymore so we can remove this completely since we have dosc in the SavedObjectsFindOptions